src/store/reducers/transfer.ts

```diff
@@ -25,13 +26,16 @@ export const transfer = combineReducers<TransferState, RootAction>({
   }),
   isTransferring: (state = false, action) => {
     switch (action.type) {
-      case getType(transferRequest): return true
+      case getType(transferRequest):
+      case getType(transactionRequest):
+        return true
 
-      case getType(transferSuccess):
-      case getType(transferFailed):
+      case getType(transactionSuccess):
```

Comment:
Are we still handling the transfer success and failure somewhere?

Reply:
I've made the transferRequest go through to the transaction caller to keep it DRY